internal/tss/store/store.go

```diff
@@ -131,3 +136,29 @@ func (s *store) GetTryByXDR(ctx context.Context, xdr string) (Try, error) {
 	}
 	return try, nil
 }
+
+func (s *store) GetTransactionsWithStatus(ctx context.Context, status tss.RPCTXStatus) ([]Transaction, error) {
+	q := `SELECT * from tss_transactions where current_status = $1`
```

Comment:
I'd suggest we follow a pattern for queries, having upper case and lower case words for SQL Statements isn't the best approach.